google-cloud-clients/google-cloud-storage/src/test/java/com/google/cloud/storage/BlobTest.java

```diff
@@ -130,6 +135,22 @@
   private static final String BASE64_KEY = "JVzfVl8NLD9FjedFuStegjRfES5ll5zc59CIXw572OA=";
   private static final Key KEY =
       new SecretKeySpec(BaseEncoding.base64().decode(BASE64_KEY), "AES256");
+  private static final RetrySettings RETRY_SETTINGS =
+      RetrySettings.newBuilder()
+          .setMaxAttempts(2)
+          .build();
+  private static final ApiClock API_CLOCK =
```

Comment:
Please document how retries are being exercised in these unit tests. It will help for posterity.